### PR TITLE
have osc new-project switch projects

### PR DESCRIPTION
**Pull request by deads2k**
When requesting a new project, it's natural to want that project added to your `openshiftconfig` and want have it selected.  This adds that functionality and fixes a message in `osc login` about requesting a project.

@smarterclayton functionality you requested.  @fabianofranz is out today, so I thought you might want to review.

### CODE REVIEW

**Comment by smarterclayton** [comment]
Add a test to verify this works to hack/test-cmd and at least verify some of the behavior here

**Comment by deads2k** [comment]
added test to test-cmd.sh

**Comment by deads2k** [comment]
@fabianofranz ptal at this one too.

File: pkg/cmd/cli/cmd/request_project.go [hunk 1]
```diff
@@ -46,8 +50,8 @@ func NewCmdRequestProject(name, fullName, oscLoginName, oscProjectName string, f
 		Short: "request a new project",
 		Long:  fmt.Sprintf(requestProjectLongDesc, fullName, oscLoginName, oscProjectName),
 		Run: func(cmd *cobra.Command, args []string) {
-			if !options.complete(cmd) {
-				return
+			if err := options.complete(cmd, f, out); err != nil {
+				glog.Fatal(err)
```
Comment (fabianofranz) [inline pkg/cmd/cli/cmd/request_project.go]: `cmdutil.CheckErr`?

Comment (smarterclayton) [inline pkg/cmd/cli/cmd/request_project.go]: Yes, also, please do a sweep of all other uses of glog.Fatal in the user facing tools.

**Comment by fabianofranz** [comment]
One comment, otherwise LGTM.

**Comment by deads2k** [comment]
@fabianofranz switched to CheckErr() and squashed in.  Also found bug in `osc login` that I've rolled in.  I'll open the upstream pull.

**Comment by deads2k** [comment]
decided I didn't like the upstream pull.  Reacting to the error instead.

**Comment by deads2k** [comment]
[merge]

**Pull request merged**